example_csv.csv

```diff
@@ -0,0 +1,7 @@
+Categorical 1,Continuous 1,Datetime 1
```

Comment:
Could you rename the file to something like `example_input.csv` so that users don't mix up the `..._csv.csv` bit

Reply:
Will need updating where referenced as well.